cd-scripts/appspec-scripts/before_install.sh

```diff
@@ -3,8 +3,10 @@
 
```

Comment:
there's also a set -e in this script but commented out, is that also because of similar reason you described in other places?

Reply:
This will be fixed! Thank you!